transcription/email.go

```diff
@@ -0,0 +1,30 @@
+package transcription
+
+import (
+	"log"
+	"net/smtp"
+	"os"
+	"strings"
+)
+
+func sendEmail(from string, to []string, subject string, body string) {
+	// Set up authentication information.
+	auth := smtp.PlainAuth(
+		"",
+		os.Getenv("MAIL_EMAIL"),
+		os.Getenv("MAIL_PASSWORD"),
+		"smtp.gmail.com",
+	)
+
+	// Connect to the server, authenticate, set the sender and recipient,
+	// and send the email all in one step.
+	msg := []byte("To: " + strings.Join(to, ", ") + "\r\n" +
+		"Subject: " + subject + "\r\n" +
+		"\r\n" +
+		body + "\r\n")
+	err := smtp.SendMail("smtp.gmail.com:25", auth, from, to, msg)
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
for reference, took most code from `smtp` example code for `SendEmail`
<img width="1201" alt="screen shot 2016-02-25 at 9 34 02 am" src="https://cloud.githubusercontent.com/assets/4250521/13322849/7d02dbb0-dba4-11e5-8643-66e887a6e1c0.png">